test/integration/react-18/test/blocking.js

```diff
@@ -20,8 +20,6 @@ export default (context, render) => {
     const $ = await get$('/suspense/thrown')
     const html = $('body').html()
     expect(html).toContain('loading')
-    expect(
-      JSON.parse($('#__NEXT_DATA__').text()).dynamicIds
-    ).not.toBeUndefined()
+    expect(JSON.parse($('#__NEXT_DATA__').text()).dynamicIds).toBeUndefined()
```

Comment:
the only core change is here, using suspense + react.lazy won't have preload ids like loadable components.